### DISCUSSION TITLE
Timers and alarms have disappeared from the official Google Home app.

### DISCUSSION THREAD

**Discussion by DurgNomis-drol** [leikoilja/ha-google-home#96] [Q&A]
I think we just developed this feature in a very timely manner. I just opened my Google Home app to remove an alarm i had set to test something and the feature is gone. I don't know if this is only for me or do any of you have the same problem?

My App version is 2.34.1.12

**Comment by leikoilja** [comment]
Oh flip, you are right, took me some time to find those in the app.
Check once a again under Audio section ![image](https://user-images.githubusercontent.com/10655107/111864641-8bc9f180-8962-11eb-8310-159f9e2675ee.png)

**Reply by DurgNomis-drol** [reply]
Thanks! I spent like 10 minutes trying to find them yesterday 😆 I was getting pretty frustrated over it

**Reply by leikoilja** [reply]
i actually saw my integration fail today in logs. Will take a closer look some of these days, but i think i have some bugs there. i m wondering if it still works for you? Google home might have changed some APIs

**Reply by DurgNomis-drol** [reply]
The only error i have gotten so far is this, but that started some days ago:

```yaml
  File "/config/custom_components/google_home/sensor.py", line 156, in _get_alarms_data
    return self.as_dict(device.get_sorted_alarms())
AttributeError: 'NoneType' object has no attribute 'get_sorted_alarms'
```

The integration will timeout sometimes. Normal at night for me.

I have had problem with my Google Nest Mini today that it wouldn't work correctly but after a restart it seems to have solved it. What errors have you gotten?

**Reply by leikoilja** [reply]
Yeh, I would get the same error as you. We ll need to try understanding what's causing it and fix it :)

**Reply by DurgNomis-drol** [reply]
I will open an issue so we can track it there :smile: #97

**Reply by leikoilja** [reply]
Thank you, @DurgNomis-drol
